src/redux/table/table.reducer.js

```diff
@@ -1,14 +1,49 @@
+import {
+  SET_TABLE_DENSE,
+  SET_CURRENT_PAGE,
+  SET_ROWS_PER_PAGE,
+  SET_PAGES_COUNT,
+  SET_ITEMS_COUNT
+} from './table.types';
+
 const initialState = {
-  dense: false
+  dense: false,
+  pagination: {
+    currentPage: 0,
+    rowsPerPage: 10,
+    rowsPerPageOptions: [10, 20, 30],
+    pagesCount: 1
+  },
+  count: 0
```

Comment:
Please, rename it to `itemsCount`, it is much easier to understand

Reply:
renamed